### PR TITLE
Added color palette result page

### PR DESCRIPTION
**Pull request by dpionk**

### CODE REVIEW

**Review by MrSpaceMan1 — APPROVED** [review]

File: skin-resque-frontend/src/pages/FrontPage/FrontPage.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ function FrontPage() {
 				</div>
 				<div className='photo-button'>
 					<img src='/images/color-test.png'></img>
-					<Button><Link to='/color-test'>Color Test</Link></Button>
+					<Button><Link to='/color-test/try'>Color Test</Link></Button>
```
Comment (kbojarowska) [inline skin-resque-frontend/src/pages/FrontPage/FrontPage.js:78]: Why do you changed this? This button should link to info not test

**Review by kbojarowska — CHANGES_REQUESTED** [review]
image
Please use % and rem values, everything is to big because of using fixed px
image
Add gap between colors

There is no save button
And what for where the arrows? They are not doing anything right now (I know that they were in figma, but what for)

**Comment by dpionk** [comment]
so for what should I include the save button? Because it wont do anything.

**Comment by kbojarowska** [comment]
> so for what should I include the save button? Because it wont do anything.

mock it, everything we do now would make future work faster, we wouldn't have to look for every page to search for missing elements, but just adding new features

**Comment by dpionk** [comment]
Then I don't understand why adding arrows is a problem

**Review by kbojarowska — APPROVED** [review]

**Pull request merged**